src/pspm_dcm.m

```diff
@@ -602,7 +602,7 @@
         % Find the position of the target trial in proc_subsessions
         last_trl_start = sbs_trlstart{i_session};
         % Convert from time (s) to data points
-        last_trl_start = last_trl_start(i_trial)*model.sr;
+        last_trl_start = ceil(last_trl_start(i_trial)*model.sr);

```

Comment:
Convert into integers for selecting the elements at the end of the array.

Reply:
@teddychao can we discuss the posted document in our next meeting? Also, can you just confirm that no trials are removed when the data available after the last trial exceeds the threshold? Thanks!